src/compile/data/filter.ts

```diff
@@ -35,6 +35,6 @@ export class FilterNode extends DataFlowNode {
   }
 
   public hash() {
-    return `Filter ${hash(this.expr)}`;
+    return `Filter ${this.expr}`;
```

Comment:
If someone passes in a long expression and we have to keep it anyway, the Js engine will just use a reference instead of making a copy. So hashing here won't buy us much. 

I thought.... But I forgot that we are making a new string. Well, expressions are usually short enough to not cause issues. It makes hashes a bit more readable so I'll leave this.